friendshipper/src-tauri/src/repo/operations/pull.rs

```diff
@@ -310,6 +312,7 @@ where
     }
 }
 
+#[instrument(skip(state))]
```

Comment:
There is some kind of sampling or erroring that's happening somewhere that's preventing this handler from sending any traces. At first I thought it impacted all `POST`s but that wound up not being true. I'd love another set of eyes on maybe some config I missed.